Loop/Managers/NotificationManager.swift

```diff
@@ -10,13 +10,15 @@ import UIKit
 import UserNotifications
 import LoopKit
 
-struct NotificationManager {
+enum NotificationManager {
```

Comment:
Strictly speaking, this isn't necessary, but it was something I had in my tree: namely, the current Loop "NotificationManager" is really not a struct, but a collection of static vars and functions.  Thus...it should never be constructed. A good practice in Swift is to make it an `enum` (oddly enough), because they are not constructable.